client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -15,9 +15,13 @@ import MediaUtils from 'lib/media/utils';
 import {
 	getImageEditorTransform,
 	getImageEditorFileInfo,
-	getImageEditorCrop
+	getImageEditorCrop,
+	imageEditorIsImageLoaded
```

Comment:
Maybe `isImageEditorImageLoaded`? I like putting the get/is at the start so you can tell what the return type is.